src/app/individual/individual-detail/individual-observation-view/individual-observation-view.component.html

```diff
@@ -1,4 +1,11 @@
-<h2>{{ 'Phänophasen' | translate }}</h2>
+<div class="individual-observation-view__title">
+  <h2>{{ 'Phänophasen' | translate }}</h2>
+  <div class="individual-observation-view__year">
```

Comment:
Is the year select font the right size as defined by Dani? Seems off to me 22px when h2 (Phänophasen above) is 18px. This applies to all of the year select dropdowns.